metadata-models/src/main/pegasus/com/linkedin/metadata/search/ChartDocument.pdl

```diff
@@ -28,15 +30,10 @@ record ChartDocument includes BaseDocument {
    */
   tool: optional string
 
-  /**
-   * Input datasets for the chart
-   */
-  inputs: optional array[DatasetUrn]
-
   /**
    * Chart query type
    */
-  queryType: optional string
+  queryType: optional ChartQueryType
```

Comment:
I'm surprised that compatibility check didn't barf at you? :)

Reply:
Thankfully, we don't have schema registration in OSS, so, it's fine :) Otherwise, yes, it would be a backward incompatible change for MSE.